hw/cdc/tools/verixcdc/run-cdc.tcl

```diff
@@ -135,12 +135,15 @@ source $CDC_WAIVER_FILE
 ## Write out report    ##
 #########################
 
-report_policy -verbose -skip_empty_summary_status -compat -output vcdc.rpt ALL
+report_policy -verbose -skip_empty_summary_status -compat -output ../REPORT/vcdc.rpt {NEW TO_BE_FIXED DEFERRED}
```

Comment:
This may mess with the Dvsim dashboard parsing mechanism - can you double check?

See here:
https://github.com/lowRISC/opentitan/blob/752c40cf2eb7d0de3eb63026a4331565865ec4f7/util/dvsim/verixcdc-report-parser.py#L46

Reply:
Reverted. Thanks!